src/content/docs/en/guides/content-collections.mdx

```diff
@@ -1,643 +1,182 @@
-
-
-```astro
----
-// src/pages/posts/[...slug].astro
-import { getEntry } from "astro:content";
-// 1. Get the slug from the incoming server request
-const { slug } = Astro.params;
-if (slug === undefined) {
-	throw new Error("Slug is required");
-}
-// 2. Query for the entry directly using the request slug
-const entry = await getEntry("blog", slug);
-// 3. Redirect if the entry does not exist
-if (entry === undefined) {
-	return Astro.redirect("/404");
-}
-// 4. (Optional) Render the entry to HTML in the template
-const { Content } = await entry.render();
----
-```
-
-:::tip
-Explore the `src/pages/` folder of the [blog tutorial demo code on GitHub](https://github.com/withastro/blog-tutorial-demo/tree/content-collections/src/pages) or [open it in StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/content-collections?file=src/pages/blog.astro) to see full examples of creating pages from your collections for blog features like a list of blog posts, tags pages, and more!
-:::
-
-## Migrating from File-Based Routing
-
-If you have an existing Astro project, such as a blog, that uses Markdown or MDX files in subfolders inside `src/pages/`, consider migrating related content or data files to content collections. 
- 
-See how to convert a basic blog example from `src/pages/posts/` to `src/content/posts` in our [step-by-step tutorial](/en/tutorials/add-content-collections/) that uses the codebase from [the Build a Blog tutorial's finished project](https://github.com/withastro/blog-tutorial-demo).
-
-## Enabling JSON Schema Generation
-
-<p><Since v="4.13.0" /></p>
-
-If you are working with collections of type `data`, Astro will auto-generate JSON schema files for your editor to get IntelliSense and type-checking. A separate file will be created for each data collection in your project based on your collections defined in `src/content/config.ts` using a library called [`zod-to-json-schema`](https://github.com/StefanTerdell/zod-to-json-schema#known-issues).
-
-This feature requires you to manually set your schema's file path as the value for `$schema` in each data entry file of the collection:
-
-```json title="src/content/authors/armand.json" ins={2}
-{
-  "$schema": "../../../.astro/collections/authors.schema.json",
-  "name": "Armand",
-  "skills": ["Astro", "Starlight"]
-}
-```
-
-Alternatively, you can set this value in your editor settings. For example, to set this value in [VSCode's `json.schemas` setting](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings), provide the path of files to match and the location of your JSON schema:
-
-```json
-{
-  "json.schemas": [
-    {
-      "fileMatch": [
-        "/src/content/authors/**"
-      ],
-      "url": "./.astro/collections/authors.schema.json"
+    ```astro ins={7} del={6}
+    // src/pages/index.astro
```

Comment:
Since we're using `getStaticPaths`, this should be a dynamic route:
```suggestion
    // src/pages/[slug].astro
```
Otherwise, looks good! No objections to merge. 😄